wagtail/core/blocks/field_block.py

```diff
@@ -594,18 +596,23 @@ class Meta:
 
 
 class PageChooserBlock(ChooserBlock):
-
-    # TODO: rename target_model to page_type
-    def __init__(self, target_model=None, can_choose_root=False,
-                 **kwargs):
+    def __init__(self, page_type=None, can_choose_root=False, target_model=None, **kwargs):
```

Comment:
Setting `page_type` as first keyword argument and moving `target_model` as last one so if the `PageChooserBlock` was created with positional argument, this should just work ™